zerver/migrations/0387_alter_stream_invite_only.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.12 on 2022-03-24 19:33
+
+from django.db import migrations, models
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def update_null_fields_to_false(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Stream = apps.get_model("zerver", "Stream")
+
+    streams = Stream.objects.filter(invite_only=None)
+
+    for stream in streams:
+        stream.invite_only = False
+        stream.save()
```

Comment:
Can we just do a single query here? This loop will be slow.

Reply:
Yep, sure.